.github/workflows/pull_request_verification.yml

```diff
@@ -20,6 +20,21 @@ jobs:
       - name: Check correct commit message
         run: ./.github/scripts/commit/checkCommitMessage.sh
 
+  count-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: ${{ github.event.pull_request.commits }}
+      - name: Count Commits
+        run: |
+          if (( ${{ github.event.pull_request.commits }} > 1 )); then
```

Comment:
`github.event.pull_request.commits` could be extracted to a variable to not repeat this

Reply:
Done